lib/msal-browser/src/config/Configuration.ts

```diff
@@ -2,14 +2,14 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
-import { AuthOptions, SystemOptions, LoggerOptions, INetworkModule, LogLevel } from "@azure/msal-common";
+import { SPAAuthOptions, SystemOptions, LoggerOptions, INetworkModule, LogLevel } from "@azure/msal-common";
 import { BrowserUtils } from "../utils/BrowserUtils";
 import { BrowserConstants } from "../utils/BrowserConstants";
 
 // Default timeout for popup windows in milliseconds
 const DEFAULT_POPUP_TIMEOUT_MS = 60000;
 
-export type BrowserAuthOptions = AuthOptions & {
+export type BrowserAuthOptions = SPAAuthOptions & {
```

Comment:
Are the 'AuthOptions` from the base class no longer applicable for `msal-node`?

Reply:
Not sure what you're asking here, could you please expand? This is browser, msal-node still use AuthOptions. Btw I haven't touched any of the configuration code, these were all changes that you made.